### ISSUE TITLE
Potential proteins

### ISSUE THREAD

**Issue by halexand** [MBL-Physiology-Bioinformatics/Phylogeny-Tutorial#1]
1. Myosins (we’ll have antibodies and tagged myosins for students to use in the lab)
2. Forkhead transcription factors, including RFX. We are currently generating antibodies against RFX proteins, but don’t have them in hand yet and don’t know if they work.  Interesting papers on RFX evolution here: https://www.pnas.org/content/107/29/12969  https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-10-130
3. Secretion/vesicular trafficking proteins (people from Lil Fritz-Laylin’s group will be in touch re: specifics)
4. Metacaspases (suggested by @Flora Rutaganira)
5. Rad51 (suggested by @MaxC )
6. CH1-CH2 domain containing proteins (e.g. utrophin, dystrophin): These tandem calponin homology domain containing proteins bind to actin, and we've found that they are sensitive to actin filament conformation.
7. Long lipid binding receptors: Both the STAB1 and STAB2 family and the TIM1, TIM3, TIM4 family of scavenger receptors have long ectodomains that interact with lipids, and the origin of lipid binding receptors would be interesting to explore.

Any others to add @djrichter ?

**Comment by halexand** [comment]
Another broad question: do we want to use hmmer based searches (seems better IMO). If so, do we want to build the profiles ahead of time? Or do we want to include that as part of the process? Could be fed into the earlier homolog searching activities (blast hmmer in session 2).

**Comment by djrichter** [comment]
Cool, thanks for sending this! I don't have any protein families to add at
the moment, although we may want to remind Lil/Flora/Max to send us their
suggestions at some point.

As it so happens, in parallel, I made the same list on the Google Doc we
had been using:
https://docs.google.com/document/d/1oAZ0igniNrAi1jKAVjfeVnNS8GfgO5PIR-JFBHz0H_Q/edit?usp=sharing

Which method do you think is a better way to keep track of the list?

**Comment by djrichter** [comment]
I agree with linking the protein family session with the BLAST/HMMer
session. That's a good idea.

Regarding whether to use HMMer or BLAST, perhaps we could do a bit of both,
in order to compare and contrast their relative strengths and weaknesses?
We could do some tests with the protein families in our list; if the
results differ between HMMer and BLAST, we could use that as an opportunity
to explore why.
